src/kirby/components/list/list-cell/list-cell.component.ts

```diff
@@ -1,12 +1,14 @@
 import { Component, OnInit, Input, HostBinding } from '@angular/core';
 
+// TODO: refactor - use map instead of enum
```

Comment:
Consider refactoring this yourself. Chances of somebody refactoring this code (later down the road) is "slim to none"

Reply:
agree - will make a new branch

Reply:
agree - will make a new branch